HTTPServer.test/ServerTests.cs

```diff
@@ -21,23 +25,39 @@ public class ServerUnitTests
         [TestMethod]
         public void ServerCanStart()
         {
-            Assert.True(server.Start() && server.Running);
+            Assert.True(server.Start(8080) != null && server.Running);
```

Comment:
If Start has already been called it will not try to start again.  It returns null if the server has already started.